nodejs/image.bzl

```diff
@@ -88,6 +86,42 @@ _dep_layer = rule(
     implementation = _dep_layer_impl,
 )
 
+def _npm_deps_runfiles(dep):
```

Comment:
add brief doc

Reply:
added a doc, does that make sense what it does?